src/unix/lwt_unix.h

```diff
@@ -24,20 +24,20 @@
 
 #define CAML_NAME_SPACE
 
-#include <lwt_config.h>
 #include <caml/mlvalues.h>
-#include <caml/unixsupport.h>
 #include <caml/socketaddr.h>
+#include <caml/unixsupport.h>
+#include <lwt_config.h>
```

Comment:
A trick is to have separate "paragraphs", clang-format will order includes by-paragraph. I've amended the commit. :)